CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ set(NETWORKIT_PYTHON "" CACHE STRING "Directory containing Python.h. Implies MON
 set(NETWORKIT_PYTHON_SOABI "" CACHE STRING "Platform specific file extension. Implies MONOLITH=TRUE")
 set(NETWORKIT_WITH_SANITIZERS "" CACHE STRING "Uses sanitizers during the compilation")
 set(NETWORKIT_RELEASE_LOGGING "AUTO" CACHE STRING "Do not compile log messages at levels TRACE or DEBUG (AUTO|ON|OFF)")
+set(NETWORKIT_RPATH "" CACHE STRING "Build specific rpath references.")
```

Comment:
This only affects the rpath of the Python module. Should we call the option `NETWORKIT_PYTHON_RPATH`?